### ISSUE TITLE
[Hunter] Pet disappeared after stabling it

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5722]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
First time I've seen this, but I saw loque'nahak and wanted to tame it. Swapped from my marksman spec to beast mastery, swapped my wolf with an old devilsaur using Call Stabled Pet, and proceeded to tame loque'nahak successfully. After the timer for Call Stabled Pet ran out, used it to stable loque'nahak - and then saw that my wolf was no longer in the stable - just an empty slot.
**Expected behaviour**: Tell us what should happen instead
I EXPECT pets placed into my stable to stay the fuck there.
**Steps to reproduce the problem**:

1. See above.
2. 
3. 

**Include proofs for this behaviour**
Wish I could, but I wasn't anticipating losing my regular wolf from a simple Call Stabled Pet swap.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rahuum** [comment]
This is still an active issue, just lost my tenacity pet after swapping away from it after Valitheria.
